doc/classes/RigidBody3D.xml

```diff
@@ -229,6 +229,7 @@
 			<description>
 				Emitted when a body enters into contact with this one. Requires [member contact_monitor] to be set to [code]true[/code] and [member contacts_reported] to be set high enough to detect all the collisions.
 				This signal not only receives the body that collided with this one, but also its [RID] ([code]body_id[/code]), the shape index from the colliding body ([code]body_shape[/code]), and the shape index from this body ([code]local_shape[/code]) the other body collided with.
+				[b]Note:[/b] Bullet physics cannot identify the shape id when using a [ConcavePolygonShape3D]. Don't use mulitple [CollisionShape3D]s when using a [ConcavePolygonShape3D] with Bullet physics.
```

Comment:
I've corrected the multiple typo and the id -> index. Thank you.

Regarding the suggested wording, I think it's better to simply tell people to not use multiple `CollisionShape3D`s when using a `ConcavePolygonShape3D` than suggesting it may work in some scenarios. The reality is, it doesn't work when using a `ConcavePolygonShape3D`.

Reply:
It could be `Don't use multiple [CollisionShape3D]s when using a [ConcavePolygonShape3D] with Bullet physics if you need shape indices.`, since it's fine when you don't care about shape indices.

Reply:
Done.